packages/realm-react/src/index.tsx

```diff
@@ -141,6 +71,87 @@ export const createRealmContext: (realmConfig?: Realm.Configuration) => RealmCon
   };
 };
 
+const defaultContext = createRealmContext();
+
+/**
+ * Returns a Context Provider component that is required to wrap any component using
+ * the Realm hooks.
+ *
+ * @example
+ * ```
+ * const AppRoot = () => {
+ *   const syncConfig = {
+ *     flexible: true,
+ *     user: currentUser
+ *   };
+ *
+ *   return (
+ *     <RealmProvider path={"data.realm"} sync={syncConfig}>
+ *       <App/>
+ *     </RealmProvider>
+ *   )
+ * }
+ * ```
+ * @param props - The {@link Realm.Configuration} for this Realm defaults to
+ * the config passed to `createRealmProvider`, but individual config keys can
+ * be overridden when creating a `<RealmProvider>` by passing them as props.
+ * For example, to override the `path` config value, use a prop named `path`
+ * e.g., `path="newPath.realm"`
+ * an attribute of the same key.
+ */
+const RealmProvider = defaultContext.RealmProvider;
```

Comment:
Might as well `export` right here?
```suggestion
export const RealmProvider = defaultContext.RealmProvider;
```

Reply:
Sure, no reason not to.